tools/solidity-coverage-example/hardhat.config.ts

```diff
@@ -0,0 +1,10 @@
+/** @type import('hardhat/config').HardhatUserConfig */
```

Comment:
Add license header to all `*.ts` and `*.js` files

Reply:
Done